src/index/mod.rs

```diff
@@ -50,6 +51,14 @@ impl Index {
         Ok(Self { path, api })
     }
 
+    pub fn run_git_gc(&self) {
+        let cmd = format!("cd {} && git gc --auto", self.path.to_str().unwrap());
+        let gc = Command::new("sh").args(&["-c", cmd.as_str()]).output();
```

Comment:
`sh` isn't available on all platforms
```suggestion
        #[cfg(not(windows))]
        const SHELL: &str = "sh";
        #[cfg(windows)]
        const SHELL: &str = "cmd";

        let gc = Command::new(SHELL).args(&["-c", cmd.as_str()]).output();
```

Reply:
I didn't see Nemo's, do what [he said](https://github.com/rust-lang/docs.rs/pull/956#discussion_r470110557)